images/Makefile

```diff
@@ -1,9 +1,7 @@
 # Copyright 2020 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-REGISTRIES ?= docker.io/cilium
-# quay.io is not enabled, see https://github.com/cilium/image-tools/issues/11
-# REGISTRIES ?= docker.io/cilium quay.io/cilium
+REGISTRIES ?= docker.io/cilium quay.io/cilium
```

Comment:
I don't think we can enable this for all images just yet, I'm not entirely sure if all images have been updated since https://github.com/cilium/image-tools/pull/59, please feel free to check as part of this.